Cargo.toml

```diff
@@ -95,16 +101,38 @@ winapi = { version = "0.3.9", features = [
     "wincon",
     "winerror",
     "winuser",
-    ]}
+] }
 # for ComPtr
 wio = { version = "0.2.2" }
 skia-safe = { version = "0.68.0", features = ["gl", "d3d", "textlayout"] }
+windows = { version = "0.56.0", features = [
+    "Win32_Security",
+    "Win32_System_LibraryLoader",
+    "Win32_System_Registry",
```

Comment:
These two redundant features (`LibraryLoader` and `Registry`) will be removed from my next PR.